pkg/ig-manager/ig-manager.go

```diff
@@ -100,6 +100,7 @@ func NewManager(runtimes []*containerutilsTypes.RuntimeConfig) (*IGManager, erro
 		containercollection.WithMultipleContainerRuntimesEnrichment(runtimes),
 		containercollection.WithContainerFanotifyEbpf(),
 		containercollection.WithTracerCollection(l.tracerCollection),
+		containercollection.WithProcEnrichment(),
```

Comment:
I think it should also be added in:
https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/gadgettracermanager/gadgettracermanager.go#L270

So it works with kubectl-gadget.